src/supermarket/config/initializers/paperclip.rb

```diff
@@ -21,7 +21,8 @@
       },
       path: path,
       bucket: ENV['S3_BUCKET'],
-      s3_protocol: ENV['PROTOCOL']
+      s3_protocol: ENV['PROTOCOL'],
+      s3_region: ENV['S3_REGION']
```

Comment:
@nathenharvey @nellshamrell Do you have opinions on this? 

Major version bump to communicate "hey, you gotta do something or behavior might change on you"?

Let it ride with a minor or patch and call the change out on the release notes and announcement?

Do more checking of S3 settings being complete?